tests/benchdnn/graph/input_displacer.cpp

```diff
@@ -196,8 +324,10 @@ int partition_data_displacer_t::displace_input_data(
 
     // do the reverse job
     auto *parent_op = &dg_->get_op_by_out_lt(tensor.id_);
+    bool backward_path_launched = false;
     while (filling_type == filling_type_t::quantization && !parent_op->empty()
             && op_ids_set_.find(parent_op->id_) != op_ids_set_.end()) {
+        backward_path_launched = true;
```

Comment:
It seems this flag is only used for printing, is it intended?

Reply:
Yes, that message is only meaningful for quantization filling. The other possibility is to introduce same message next to each break. I decided to stick to this one.